### PR TITLE
[Chore] Add test for verifying that log files are included in debug reports

### PR DESCRIPTION
**Pull request by typfel**
## What's new in this PR?

Add test for verifying that log files are included in debug reports

### CODE REVIEW

File: Wire-iOS Tests/LoggingTest.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import XCTest
+import WireSystem
+
+class LoggingTest: XCTestCase {
+
+    func testThatCurrentLogIsNotEmpty() {
+        // Assert that some logs have been written to the current log file, which is the file
+        // which gets attached to debug reports.
+        XCTAssertNotNil(ZMSLog.currentLog)
```
Comment (johnxnguyen) [inline Wire-iOS Tests/LoggingTest.swift]: How does this assert that logs have been written to the file?

Comment (typfel) [reply Wire-iOS Tests/LoggingTest.swift]: It doesn't :) 

I'll  add an additional assert that the file is not empty.

Comment (johnxnguyen) [reply Wire-iOS Tests/LoggingTest.swift]: Is that coming in this PR or another?

Comment (typfel) [reply Wire-iOS Tests/LoggingTest.swift]: No added it now.

**Pull request merged**